### ISSUE TITLE
Multiple GUI problems

### ISSUE THREAD

**Issue by vgturtle127** [piorrro33/psxjin#60]
I will supply some annotated screenshots to show what I'm talking about.

![screenshot 22edited](https://cloud.githubusercontent.com/assets/3183676/8635862/b2320928-2803-11e5-9212-bb9876f2fa72.jpg)

First of all, the Open CD/Close CD/Eject/Open should all be combined into one option. As far as I know, they all have the same functionality. Basically, they open and close the emulated CD tray. So, if the user ever wanted to open or close said CD tray, it should be one option because if the CD is open, they won't want to open it again. That would just be silly. :smile: 

Lua scripting might belong better in the tools section, as well as the screenshot and movie functions.

![screenshot 23edited](https://cloud.githubusercontent.com/assets/3183676/8635863/b23295d2-2803-11e5-957d-99a308481fc0.jpg)

Again, the CD options should be combined into one.

I'm not sure if the analog control option is part of the input plugin and they just added it because the plugin had it, but we should investigate as to whether it is really appropriate to have it here and not in the controller options.

The "display this" options are all very buggy and when they do work, they are really only useful to certain use case scenarios. We should probably put these in with tools. Along with fixing them, of course. :smile: 

![screenshot 24edited](https://cloud.githubusercontent.com/assets/3183676/8635865/b2358b16-2803-11e5-98a6-168f375e4e0e.jpg)

BIOS should be in all caps since it is an acronym.

Graphics to memory cards is perfect. Map hotkeys should maybe be in the controller options? Or maybe elsewhere, like in tools. We should re-evaluate it, regardless.

Options may be relocated to emulator, as it has options that concern proper use (fixes for certain games, region detection, enabling the console log output, etc.

![screenshot 25edited](https://cloud.githubusercontent.com/assets/3183676/8635864/b2353134-2803-11e5-8f01-df425fe5085c.jpg)

Online help should probably re-locate to our Wiki. About needs to be updated along with the README (I will do the README, and I will try and update the About page soon).

![screenshot 26edited](https://cloud.githubusercontent.com/assets/3183676/8635866/b2361888-2803-11e5-9fa2-08040789a4ac.jpg)

We need to update how the version works. The stable versions should not have an SVN build number (especially because we are using :octocat: Git :octocat:  now :smile: ). The development version should have some sort of indicator that they are not stable builds (most likely 2.0.3dev or something along those lines).

Also, we should make it so the user can change the size of the window manually, and have the window adjust automatically to the users window. Many emulators do this without sacrificing anything, so I think investing the time will be worth it, especially for people who play for a long time.

We should probably think about making some changes to the menu as a whole. I kind of feel like the menu for the most part is very redundant. Tools are scattered across multiple menu options, various debug features are as well, many options that lead to the same thing, etc. Making the GUI that much better will make it easier for users to navigate, therfore easier to use and faster for finding bugs, recording, playing, having fun, etc. Which is all good. :smiley: 

These are not things we can fix in a day, but I think if all these changes are made we will have an emulator that is far better than it's previous versions. So much so that people might choose it over PCSX-Reloaded, PCSX-RR, or ePSXe! That is a crazy idea, but hey. I'm crazy :smile: 

It already has benefits over other emulators, so might as well try to make it the best we can. :smile:

**Comment by piorrro33** [comment]
"The development version should have some sort of indicator that they are not stable builds" That's why I named these versions v2.0.3pre.
I agree for the other things, but all you are talking in this issue is not urgent so that's not the first thing we should patch/do.

"So much so that people might choose it over PCSX-Reloaded, PCSX-RR, or ePSXe!" First we'll have to get a better compatibility than them and after that yes. ^^

**Comment by vgturtle127** [comment]
I agree with it not being urgent, and actually I already fixed some of the things. :smile: 

It's kind of funny that you mention compatibility, since it seems like the best emulator changes with every game. Kind of sucks, really. :disappointed:

**Comment by piorrro33** [comment]
Well, TASers use PSXJin only because it has TAS input, if ePSXe for example had TAS input, I'm pretty sure no one would use PSXJin because of its poor compatibility. (at least with the games I tested)

**Comment by vgturtle127** [comment]
Poor compatibility, you say?

*Dashes off to code more*

**Comment by piorrro33** [comment]
Well, before coding more, you probably should push your changes to the code so that I could help you. :smile:

**Comment by vgturtle127** [comment]
Yeah, I know. I am almost done. 3 more files to go!

They are huge though.

I decided not to touch the DirectX libraries until I can ensure that PSXjin compiles and runs.

But I swear I'm almost done this time! :laughing:

**Comment by piorrro33** [comment]
Great!
